test/test_uri_utils.rb

```diff
@@ -47,14 +47,16 @@ def test_split_file_uri
     parts = split_file_uri("file:///usr/local/var/github/app/assets/javascripts/application.js")
     assert_equal ['file', nil, '/usr/local/var/github/app/assets/javascripts/application.js', nil], parts
 
-    parts = split_file_uri("file:///C:/Documents%20and%20Settings/davris/FileSchemeURIs.doc")
-    assert_equal ['file', nil, 'C:/Documents and Settings/davris/FileSchemeURIs.doc', nil], parts
+    if DOSISH
```

Comment:
Why we need this conditional now?

Reply:
```file:///C:/``` is not a valid uri on linux

Reply:
Right, but this test was running and passing on linux, so why add the conditional now?

Reply:
the previous version always stripped ```C:``` from path
```
path.gsub!(/^\/([a-zA-Z]:)/, '\1'.freeze)
```
now we can eliminate this line, because these paths shouldn't exist on nix systems.

windows
```
file:///C:/Documents
```
vs nix
```
file:///usr/local
```

I need to add guards for these tests. On windows it's still stripped but in a more effective way
```
path = path[1..-1]
```